binaries/runtime/src/operator/python.rs

```diff
@@ -170,24 +169,22 @@ pub fn spawn(
         Result::<_, eyre::Report>::Ok(reason)
     };
 
-    thread::spawn(move || {
```

Comment:
With this change, the `spawn` function blocks until the operator finishes, doesn't it? So spawning multiple operators will executate them sequentially, and also block the runtime.

Reply:
Yes. Basically, to make it work. I have to not spawn the python operator. And instead run the python operator in the main thread. I had to make the main thread that used to contain the "stopping thread" spawn in another thread. This ultimately makes the runtime only usable with one python operator. 

This only applies for python operator. You can spawn other rust/c operator before python for hybrid runtime